### ISSUE TITLE
Documentation does not describe implicit `project.clj` options

### ISSUE THREAD

**Issue by swannodette** [tomjakubowski/weasel#53]
```
:repl-options {:nrepl-middleware [cemerick.piggieback/wrap-cljs-repl]}
```

is necessary for the README instructions to work at all.

**Comment by tomjakubowski** [comment]
Yeah, I kind of handwaved much of this away with a one sentence instruction to "set up Piggieback," which would include that. The README should describe that process in more detail.

**Comment by MalloZup** [comment]
autogenerated with https://github.com/MalloZup/doghub:  issue inactive since 450 days. Please update the issue or close it
